### DISCUSSION TITLE
[6 settembre 2023] Complessità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#18] [Complessità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/dd23a93c-ec1d-4f77-9b5b-a5613b909149)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/7d917f1d-2243-4669-a23f-2a552456caa1)

----

Per quanto riguarda la seconda dimostrazione, poiché $SAT \in \textsf{NP-Complete}$, è sufficiente mostrare che $3COL \in \textsf{NP}$:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/7744d0fc-9129-410a-800f-7686baa1fa08)
